app/models/user.rb

```diff
@@ -0,0 +1,25 @@
+class User < ApplicationRecord
+  VALIDATE_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  has_secure_password
+  before_save :down_case
```

Comment:
```suggestion
  before_save :downcase
```
ri cụ, tự dưng tách chữ của ngừ ta ra là seo